app/src/main/java/com/example/isioyemohammed/gitlist/ListItem.java

```diff
@@ -0,0 +1,25 @@
+package com.example.isioyemohammed.gitlist;
```

Comment:
@codestaintin 
Should this file be part of this PR. 

The PR title is *Create layout file for RecyclerView*

Reply:
@TheDancerCodes , thanks for the feedback, duly noted!

Reply:
Great! 👍 @codestaintin